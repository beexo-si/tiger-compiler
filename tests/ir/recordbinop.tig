let
    type l = {elem:int}
    var ls:l := l{elem=1}
in
    if(ls>ls) then 1 else 0
end
