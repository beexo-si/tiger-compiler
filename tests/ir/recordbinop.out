Fail: Error -- l√≠nea 5: Error de tipos
